code/ui/manager/src/components/preview/preview.tsx

```diff
@@ -90,15 +92,21 @@ const createCanvas = (id: string, baseUrl = 'iframe.html', withLoader = true): A
           const isLoading = entry
             ? refLoading || rootLoading
             : (!storiesFailed && !storiesConfigured) || rootLoading;
+          const isBooting = !storySpecified && !storiesFailed;
 
+          console.log({ withLoader, isLoading, isBooting, progress });
           return (
             <ZoomConsumer>
               {({ value: scale }) => {
                 return (
                   <>
-                    {withLoader && isLoading && (
+                    {withLoader && (isLoading || isBooting) && (
                       <S.LoaderWrapper>
-                        <Loader id="preview-loader" role="progressbar" progress={progress} />
+                        <Loader
+                          id="preview-loader"
+                          role="progressbar"
+                          progress={progress || { value: 1, message: 'Booting' }}
```

Comment:
```suggestion
                          progress={progress || { value: 1, message: 'Loading' }}
```

Wouldn't "Loading" be more user friendly? Or "Fetching modules" would be more precise I guess in this instance, but that might be too much.

I just feel like Booting is something that an OS or server does, not really what our manager is doing here.